make/dev.mk

```diff
@@ -25,10 +26,15 @@ dev-deploy-e2e-two-members: deploy-e2e-to-dev-namespaces-two-members print-reg-s
 .PHONY: deploy-e2e-to-dev-namespaces
 deploy-e2e-to-dev-namespaces:
 	$(MAKE) deploy-e2e MEMBER_NS=${DEV_MEMBER_NS} SECOND_MEMBER_MODE=false HOST_NS=${DEV_HOST_NS} REGISTRATION_SERVICE_NS=${DEV_REGISTRATION_SERVICE_NS} ENVIRONMENT=${DEV_ENVIRONMENT} E2E_TEST_EXECUTION=false IS_OSD=${IS_OSD} DEPLOY_LATEST=${DEPLOY_LATEST}
+	if [[ ${DEV_SSO} == true ]]; then \
+		$(MAKE) download-assets ASSETS_FOLDER=scripts/ci/dev-sso; \
+		$(MAKE) run-cicd-script SCRIPT_PATH=scripts/ci/setup-dev-sso.sh  SCRIPT_PARAMS="--sso-ns $(DEV_SSO_NS)"; \
+	fi
```

Comment:
looking at this, I'm wondering, wouldn't it be easier to move the assets and the script into toolchain-e2e and execute only the `wait-until-is-installed.sh` from toolchain-cicd repo? I guess that this is the only reason for putting it into the toolchain-cicd repo, right?

Reply:
this is one reason indeed, the other reason is that we have all the bash scripts in toolchain-cicd and they are consumed by toolchain-e2e, so I've just followed the convention here. 

I'm fine with moving them but I don't think it will simplify much. Plus the download assets methods seems to be working fine.

Reply:
Yeah, I understand that you wanted to follow the same format.
Technically speaking, we could move the whole `ci/` folder into toolchain-e2e repo, I don't see any significant reason for keeping them there. IIRC, they are there for the historical reasons - some of them were called from multiple repos.

Anyway, if you don't think that it would simplify anything, then feel free to keep it as it is.

Reply:
It makes sense to me to move them all, if this doesn't break anything ( I'm thinking about konflux testing for example ) . I think moving them all would simplify, moving just this one not that much IMO. So if it's ok for you let's do it in a refactoring follow up PR and simplify the make targets here as well.